### PR TITLE
Refine the description of destroy command to be more accurate

### LINKED ISSUE #9901: pulumi cli description for destroy is wrong
pulumi cli description for destroy is wrong
[internal slack convo](https://pulumi.slack.com/archives/C85BS3LJZ/p1655230372394299)

file to fix up: https://github.com/pulumi/pulumi/blob/master/pkg/cmd/pulumi/destroy.go#L63-L70

### PR DESCRIPTION
**Pull request by mikhailshilkov**
# Description

Fixes #9901

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works

- [ ] I have updated the [CHANGELOG-PENDING](https://github.com/pulumi/pulumi/blob/master/CHANGELOG_PENDING.md) file with my change

- [ ] Yes, there are changes in this PR that warrants bumping the Pulumi Service API version

### CODE REVIEW

File: pkg/cmd/pulumi/destroy.go [hunk 1]
```diff
@@ -60,13 +60,15 @@ func newDestroyCmd() *cobra.Command {
...
 		SuggestFor: []string{"delete", "kill", "remove", "rm", "stop"},
-		Short:      "Destroy an existing stack and its resources",
-		Long: "Destroy an existing stack and its resources\n" +
+		Short:      "Destroy all the existing resources in the stack",
+		Long: "Destroy all the existing resources in the stack\n" +
 			"\n" +
-			"This command deletes an entire existing stack by name.  The current state is\n" +
+			"This command deletes all the resources in the selected stack.  The current state is\n" +
 			"loaded from the associated state file in the workspace.  After running to completion,\n" +
 			"all of this stack's resources and associated state will be gone.\n" +
```
Comment (Frassle) [inline pkg/cmd/pulumi/destroy.go:68]: Wonder if this should be rephrased "all ... associated state" sounds like all the state for the stack, but it's not its just all the state for the resources.

File: pkg/cmd/pulumi/destroy.go [hunk 2]
```diff
@@ -60,13 +60,15 @@ func newDestroyCmd() *cobra.Command {
 		Use:        "destroy",
 		Aliases:    []string{"down"},
 		SuggestFor: []string{"delete", "kill", "remove", "rm", "stop"},
-		Short:      "Destroy an existing stack and its resources",
-		Long: "Destroy an existing stack and its resources\n" +
+		Short:      "Destroy all the existing resources in the stack",
+		Long: "Destroy all the existing resources in the stack\n" +
 			"\n" +
-			"This command deletes an entire existing stack by name.  The current state is\n" +
+			"This command deletes all the resources in the selected stack.  The current state is\n" +
```
Comment (susanev) [inline pkg/cmd/pulumi/destroy.go:66]:
```suggestion
			"Deletes all the resources in the selected stack.  The current state is\n" +
```

File: pkg/cmd/pulumi/destroy.go (hunk 1 above)
Comment (susanev) [inline pkg/cmd/pulumi/destroy.go:68]:
```suggestion
			"all of this stack's resources and associated state are deleted.\n" +
```

File: pkg/cmd/pulumi/destroy.go [hunk 3]
```diff
@@ -60,13 +60,15 @@ func newDestroyCmd() *cobra.Command {
 		Use:        "destroy",
 		Aliases:    []string{"down"},
 		SuggestFor: []string{"delete", "kill", "remove", "rm", "stop"},
-		Short:      "Destroy an existing stack and its resources",
-		Long: "Destroy an existing stack and its resources\n" +
+		Short:      "Destroy all the existing resources in the stack",
+		Long: "Destroy all the existing resources in the stack\n" +
```
[the remark this answers is not in the document]
Comment (susanev) [reply pkg/cmd/pulumi/destroy.go:64]: thoughts on adding this at the top as well?

**Review by susanev — APPROVED** [review]

File: pkg/cmd/pulumi/destroy.go [hunk 4]
```diff
@@ -60,13 +60,15 @@ func newDestroyCmd() *cobra.Command {
...
 			"\n" +
-			"This command deletes an entire existing stack by name.  The current state is\n" +
+			"This command deletes all the resources in the selected stack.  The current state is\n" +
 			"loaded from the associated state file in the workspace.  After running to completion,\n" +
 			"all of this stack's resources and associated state will be gone.\n" +
 			"\n" +
+			"The stack itself is not deleted. Use `pulumi stack rm` to delete the stack.\n" +
+			"\n" +
 			"Warning: this command is generally irreversible and should be used with great care.",
```
Comment (susanev) [inline pkg/cmd/pulumi/destroy.go:72]: what does generally irreversible mean? if it means irreversible we should say only that and not generally. if there's some exception we should say what the exception is.

File: pkg/cmd/pulumi/destroy.go (hunk 1 above)
Comment (mikhailshilkov) [reply pkg/cmd/pulumi/destroy.go:68]: Not sure I get this - what's your suggested wording?

File: pkg/cmd/pulumi/destroy.go (hunk 4 above)
Comment (mikhailshilkov) [reply pkg/cmd/pulumi/destroy.go:72]: I guess it means that some resources may be recreated with no harm, while others may be lost forever (e.g. data in a database).

File: pkg/cmd/pulumi/destroy.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (mikhailshilkov) [reply pkg/cmd/pulumi/destroy.go:64]: I feel like it's better as it is...

[the remark this answers is not in the document]
Comment (susanev) [reply pkg/cmd/pulumi/destroy.go]: Okay, let's keep it as is then. If more folks report confusion though we should consider raising the prominence of this detail.

**Pull request merged**